data_gateway/configuration.py

```diff
@@ -1,188 +1,241 @@
+    :param float baros_freq: barometers sampling frequency
+    :param float baros_bm: TODO nobody seems to know...
+    :param float diff_baros_freq: differential barometers sampling frequency
+    :param string blade_id: The id of the blade on which the node is mounted, if known
     :param float constat_period: period of incoming connection statistic parameters in ms
-    :param int serial_buffer_rx_size: serial receiving buffer size in bytes
-    :param int serial_buffer_tx_size: serial transmitting buffer size in bytes
-    :param float baudrate: serial port baud rate
-    :param Literal["little", "big"] endian: one of "little" or "big"
-    :param float max_timestamp_slack: TODO   # 5ms
+    :param float gyro_freq: gyrometers sampling frequency
+    :param float gyro_range: TODO nobody seems to know...
     :param float max_period_drift: TODO   # 2% difference between IMU clock and CPU clock allowed
-    :param int packet_key: TODO
+    :param float max_timestamp_slack: TODO   # 5ms
+    :param float mics_bm: TODO nobody seems to know...
+    :param float mics_freq: microphones sampling frequency
+    :param string node_firmware_version: The verison of the firmware on the node, if known.
     :param int type_handle_def: TODO
-    :param int mics_samples_per_packet: number of samples per packet from microphones
-    :param int imu_samples_per_packet: TODO
-    :param int analog_samples_per_packet: number of samples per packet from analog sensors
-    :param int baros_samples_per_packet: number of samples per packet from barometers
-    :param int constat_samples_per_packet: number of samples per packet from connection statistics
-    :param list sensor_names: Sensors present on the measurement node
-    :param dict|None default_handles: Each handle identifies packet type, i.e. expected contents of packet payload
-    :param dict|None samples_per_packet: TODO
-    :param dict|None number_of_sensors: TODO
-    :param dict|None period:
+    :param dict|None default_handles: Map of the default handles which a node will use to communicate packet type (the expected contents of packet payload). These are defaults, as they may be altered on the fly by a node.
+    :param dict|None number_of_sensors: A map for each sensor, giving the number of samples expected from that sensor
+    :param dict|None samples_per_packet: A map for each sensor, giving the number of samples sent in a packet from that sensor
     :param dict|None sensor_commands:
-    :param dict|None installation_data: metadata about the current session of the gateway provided by the user
+    :param list|None sensor_names: List of sensors present on the measurement node
     :return None:
     """
 
     def __init__(
         self,
-        mics_freq=15625,
-        mics_bm=0x3FF,
-        baros_freq=100,
-        diff_baros_freq=1000,
-        baros_bm=0x3FF,
         acc_freq=100,
         acc_range=16,
+        analog_freq=16384,
```

Comment:
As we're making breaking changes, shall we rename all the parameter names in these constructors to use full words instead of abbrevations? e.g. what does `baros_bm` mean?